packages/kbn-shared-ux-components/src/toolbar/buttons/add_from_library/add_from_library.mdx

```diff
@@ -0,0 +1,12 @@
+---
+id: sharedUX/Components/AddFromLibraryButton
+slug: /shared-ux/components/toolbar/buttons/add_from_library
+title: Add From Library Button
+summary: An example of the primary button
+tags: ['shared-ux', 'component']
+date: 2022-02-17
```

Comment:
nit: update date

Reply:
Cool! Updated in [63d8c31](https://github.com/elastic/kibana/pull/127605/commits/63d8c31f0de1ff81e9cff47f29178a03f01fc9d7) thanks!